src/utils/mediaSource.ts

```diff
@@ -6,6 +6,9 @@ import type { MediaSourceInfo } from '@jellyfin/sdk/lib/generated-client';
  * @returns _true_ if the media source is an HLS stream, _false_ otherwise.
  */
 export function isHls(mediaSource: MediaSourceInfo|null|undefined): boolean {
-    const protocol = mediaSource?.TranscodingSubProtocol || mediaSource?.Container;
-    return protocol?.toUpperCase() === 'HLS';
+    if (mediaSource?.TranscodingSubProtocol?.toUpperCase() === 'HLS') {
+        return true;
+    }
+
+    return mediaSource?.Container?.toUpperCase() === 'HLS';
```

Comment:
_It is much more convenient to view the code from a PC rather than a mobile._ :sweat_smile: 
```suggestion
    return mediaSource?.TranscodingSubProtocol?.toUpperCase() === 'HLS'
        || mediaSource?.Container?.toUpperCase() === 'HLS';
```